devsetup/roles/download_tools/tasks/main.yaml

```diff
@@ -1,61 +1,75 @@
+---
 - name: Install build dependencies
   become: true
   become_user: root
-  package:
+  ansible.builtin.package:
     name:
       - jq
       - skopeo
       - sqlite
 
 - name: Set opm download url suffix
-  set_fact: opm_url_suffix="latest/download"
+  ansible.builtin.set_fact:
+    opm_url_suffix="latest/download"
   when: opm_version is undefined or opm_version == "latest"
 
 - name: Set opm download url suffix
-  set_fact: opm_url_suffix="download/{{ opm_version }}"
+  ansible.builtin.set_fact:
+    opm_url_suffix="download/{{ opm_version }}"
   when: opm_version is defined and opm_version != "latest"
 
 - name: Create $HOME/bin dir
-  file:
+  ansible.builtin.file:
     path: "{{ lookup('env', 'HOME') }}/bin"
     state: directory
     mode: '0755'
 
 - name: Download opm
-  get_url:
-    url: https://github.com/operator-framework/operator-registry/releases/{{ opm_url_suffix }}/linux-amd64-opm
+  ansible.builtin.get_url:
+    url:
+      "https://github.com/operator-framework/operator-registry/releases/\
+      {{ opm_url_suffix }}/linux-amd64-opm"
     dest: "{{ lookup('env', 'HOME') }}/bin/opm"
     mode: '0755'
     timeout: 30
 
 - name: Get version from sdk_version
-  set_fact: _sdk_version="{{ sdk_version | regex_search('v(.*)', '\\1') | first }}"
+  ansible.builtin.set_fact:
+    _sdk_version="{{ sdk_version | regex_search('v(.*)', '\\1') | first }}"
 
 - name: Set operator-sdk file for version < 1.3.0
-  set_fact: _operator_sdk_file="operator-sdk-{{ sdk_version }}-x86_64-linux-gnu"
+  ansible.builtin.set_fact:
+    _operator_sdk_file="operator-sdk-{{ sdk_version }}-x86_64-linux-gnu"
   when: _sdk_version is version('1.3.0', 'lt', strict=True )
 
 - name: Set operator-sdk file for version >= 1.3.0
-  set_fact: _operator_sdk_file="operator-sdk_linux_amd64"
+  ansible.builtin.set_fact:
+    _operator_sdk_file="operator-sdk_linux_amd64"
   when: _sdk_version is version('1.3.0', 'ge', strict=True )
 
 - name: Download operator-sdk
-  get_url:
-    url: https://github.com/operator-framework/operator-sdk/releases/download/{{ sdk_version }}/{{ _operator_sdk_file }}
+  ansible.builtin.get_url:
+    url: >
+      https://github.com/operator-framework/operator-sdk/releases/download/
+      {{ sdk_version }}/{{ _operator_sdk_file }}
```

Comment:
IIRC, `>` add a newline at the end, you should prefer `>-`

Reply:
I've tried `>` and `>=` with ansible, [results](https://github.com/openstack-k8s-operators/install_yamls/pull/68#discussion_r1091475504) shows invalid urls. Please check. 

CC: @rebtoor @jpodivin